docs/catalogue/cat_cohort-data-manager.md

```diff
@@ -4,16 +4,15 @@
 
 [MOLGENIS Data Catalogue](https://data-catalogue.molgeniscloud.org/catalogue/catalogue/#) provides a
 framework to describe in detail: metadata of cohorts and of data sources; definitions of the variables collected in
-cohorts and in the data banks composing the data sources (aka 'source variables'); and mappings to common data models 
-(aka 'target variables'). Its purpose is to facilitate pooled data analysis of multiple cohorts together 
-[Fortier et al, 2017](https://pubmed.ncbi.nlm.nih.gov/27272186/), and multi-data
+cohorts and data sources; and mappings to common data models. Its purpose is to facilitate pooled data analysis of multiple cohorts
+[Fortier et al, 2017](https://pubmed.ncbi.nlm.nih.gov/27272186/) and multi-data
 source studies [Gini et al, 2020](https://pubmed.ncbi.nlm.nih.gov/32243569/).
 
 - The metadata of cohorts include descriptive information such as contact details, name of the cohort, and high-level
   summary of contents and cohort design. The metadata of data sources, of the corresponding data banks and of the
   institutions that provide access to them, include descriptive information such as contact details, reason for
```

Comment:
do we need to rename institutions to organisations (datamodel 3)?

Reply:
Organisations is a bit more broad then Institutions, you can fill out departments, companies, research institutes etc.